i18n/en/docusaurus-plugin-content-docs/current/get-started/tutorial.mdx

```diff
@@ -0,0 +1,283 @@
+![Two columns of items connected by lines. On the left are entities, on the right are applications. The applications are: a pizza shop application, a cryptocurrency exchange, a social network, and our note app. The Order entity is connected to the pizza shop and the cryptocurrency exchange. The Meal entity is connected to the pizza shop. The User entity is connected to all 4 applications. The Book entity is not connected to any application. The Comment entity is connected to the social network. The Currency entity is connected to the cryptocurrency exchange. The Note entity is connected to our note app.](/img/tutorial/business-domains-light.jpg#light-mode-only)
+![Two columns of items connected by lines. On the left are entities, on the right are applications. The applications are: a pizza shop application, a cryptocurrency exchange, a social network, and our note app. The Order entity is connected to the pizza shop and the cryptocurrency exchange. The Meal entity is connected to the pizza shop. The User entity is connected to all 4 applications. The Book entity is not connected to any application. The Comment entity is connected to the social network. The Currency entity is connected to the cryptocurrency exchange. The Note entity is connected to our note app.](/img/tutorial/business-domains-dark.jpg#dark-mode-only)
+
+As we can see, a note is certainly a part of our business domain, while a meal certainly isn’t, even though notes can contain dinner recipes. A user is also a part of our business domain, which is true for all applications that have some sort of authorization. Since our app is pretty simple, these two entities alone make up our entire business domain. It’s a pretty small domain.
+
+As you might remember from the [overview], Feature-Sliced Design has a layer called Entities. This layer is entirely devoted to entities from the business domain — one slice for each entity. Therefore, in our future app, the folder `entities/` will have two subfolders, `note/` and `user/`.
+
+```
+📂 entities/
+  📁 note/
+  📁 user/
+```
+
+### Squeezing business value out of entities
+
+Entities by themselves don’t produce any business value — they simply exist, waiting for you to do something with them. If you recall, we previously wrote down a list of things we can do in our app:
+
+- read a note
+- create/edit a note
+- delete a note
+- synchronise your notes across devices
+
+This list is all about notes, but our business domain also includes a user. That’s how we know that this list is incomplete. It must also include actions on the user entity, namely:
+
+- log in, or present yourself as a certain user
+- log out, or revoke access to a user’s content on a device
+
+The things we write in this list really are just features of our application, only described in excruciating detail. Good thing that Feature-Sliced Design also has a layer called Features! That’s where these will go:
+
+```
+📂 features/
+  📁 edit-note/
+  📁 delete-note/
+  📁 sync-notes/
```

Comment:
I didn't find any button or something in images which references `sync-notes` feature, so I don't get why we need this feature